VMEncryption/setup.py

```diff
@@ -81,11 +81,11 @@
   "name": CommonVariables.extension_name,
   "version": "1.0",
   "handlerManifest": {
-    "installCommand": main_entry + " -install",
-    "uninstallCommand": main_entry + " -uninstall",
-    "updateCommand": main_entry + " -update",
-    "enableCommand": main_entry + " -enable",
-    "disableCommand": main_entry + " -disable",
+    "installCommand": "extension_shim.sh -c {0} --install".format(main_entry),
```

Comment:
>extension_shim.sh [](start = 23, length = 17)

are there scenarios where shell script execution is not allowed on the system?

Reply:
The problem being addressed here is that the extension did not gracefully with an "unsupported" error message and the proper exit code for telemetry when installed on a custom image without the necessary python 2.7 support present.   Even on a system that cannot execute python scripts, it can still usually support a shell script.  By adding this shim, we can fail gracefully from within the shim on systems that do not support python.